a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -278,3 +300,19 @@ def get_rack_vthunder_create_member_flow(self):
                                    requires=(constants.LOADBALANCER,
                                              constants.LISTENERS)))
         return create_member_flow
+
+    def _get_create_member_vlan_network_subflow(self):
+        vlan_network_subflow = linear_flow.Flow(a10constants.CREATE_VLAN_NETWORK_SUBFLOW)
+        vlan_network_subflow.add(a10_network_tasks.GetMemberSubnetVLANID(
```

Comment:
This should be compressed into the CreateVlan task

Reply:
Done. Very good comments. The whole code got compressed to a task.